src/data/dapp_promotions.json

```diff
@@ -52,5 +52,11 @@
     "shortDescription": "#1 Cross-chain Social Fitness App on iOS & Android, Apple Watch",
     "link": "https://app.moonfit.xyz/astar-reward",
     "img": "images/dapp_promotions/moonfit.png"
+  },
+  {
+    "name": "Ledger - dApp Staking V3",
```

Comment:
Put the promotion info at the beginning of the file otherwise it won't be shown, because we are showing only first 5 promotions